.github/workflows/ci.yaml

```
@@ -41,17 +41,22 @@ jobs:
         df -h
         ulimit -a
 
-    - name: Environment Information
+    - name: Install topology refactor branch
       shell: bash -l {0}
       run: |
-        conda info
-        conda list
+        pip install git+https://github.com/openforcefield/openff-toolkit.git@topology-biopolymer-refactor
```

For today's working session:

```suggestion
        pip install git+https://github.com/openforcefield/openff-toolkit.git@topology_molecule_deprecation
```